### DISCUSSION TITLE
Why aren't the Docker images published to Dockerhub or similar?

### DISCUSSION THREAD

**Discussion by Dan6erbond** [padloc/padloc#582] [Q&A] [answered]
I just tried to spin up the PadLoc services (PWA and server) using Docker compose, and until I came across the way this repository references the github.com `Dockerfile-server` and `Dockerfile-pwa` images, I noticed that it was spinning up a completely wrong (maybe old/outdated) container that had some issues.

This not only proposes a security risk, but it makes PadLoc incompatible with Kubernetes deployments and modern container environments. It requires users to build their own images, which is more advanced, and would also make deployment instructions a lot easier, if people could just fetch the containers from a public source.

**Comment by MaKleSoft** [comment]
Thanks for reminding us! The official docker images are indeed embarrassingly outdated. I'll create an issue for this.

**Reply by Dan6erbond** [reply]
Any ETA on how quickly this can be resolved? I wouldn't mind providing support, if needed, to add a CI pipeline so that future releases are also properly versioned and released to Dockerhub. Especially for security reasons!

**Reply by MaKleSoft** [reply]
@BrunoBernardino CI is your expertise :) Can you treat this with priority?

**Reply by BrunoBernardino** [reply]
Alright, being tracked at https://github.com/padloc/padloc/issues/583

**Reply by BrunoBernardino** [reply]
Will be fixed by v4.1.0

**Comment by BrunoBernardino** [comment, marked as answer]
Will be fixed by v4.1.0, closed by #586
